util/chplenv/chpl_unwind.py

```diff
@@ -12,6 +12,10 @@ def get():
     osx = platform_val.startswith('darwin')
     val = overrides.get('CHPL_UNWIND')
 
+    if val and val != 'none' and val != 'bundled' and val != 'system':
+        error("Invalid CHPL_UNWIND value {0}\n"
+              "Valid values are none, bundled, or system".format(val))
+
```

Comment:
I think writing this as `val and val not in ('none', 'bundled', 'system')` is cleaner. 

Although you could make `val` default to `'none'` and remove the `val and` part, but thats a bigger change that requires tweaking the if-statement at the end